recommendations/guides/03_my_interests.adoc

```diff
@@ -89,127 +89,6 @@ In this query we
 This is quite a crude scoring mechanism which favours generally popular topics.
 We'll have a look at how to address that later on.
 
-== What if I don't specify any interests?
```

Comment:
Need to revisit this section, figure out how to make more performant for larger Meetup datasets